services/thermal_mgr/thermal_mgr.c

```diff
@@ -24,48 +24,95 @@ static uint8_t thermalMgrQueueStorageArea[THERMAL_MGR_QUEUE_LENGTH * THERMAL_MGR
 
 static void thermalMgr(void *pvParameters);
 
-void initThermalSystemManager(lm75bd_config_t *config) {
+void initThermalSystemManager(lm75bd_config_t *config)
+{
   memset(&thermalMgrTaskBuffer, 0, sizeof(thermalMgrTaskBuffer));
   memset(thermalMgrTaskStack, 0, sizeof(thermalMgrTaskStack));
-  
+
   thermalMgrTaskHandle = xTaskCreateStatic(
-    thermalMgr, "thermalMgr", THERMAL_MGR_STACK_SIZE,
-    config, 1, thermalMgrTaskStack, &thermalMgrTaskBuffer);
+      thermalMgr, "thermalMgr", THERMAL_MGR_STACK_SIZE,
+      config, 1, thermalMgrTaskStack, &thermalMgrTaskBuffer);
 
   memset(&thermalMgrQueueBuffer, 0, sizeof(thermalMgrQueueBuffer));
   memset(thermalMgrQueueStorageArea, 0, sizeof(thermalMgrQueueStorageArea));
 
   thermalMgrQueueHandle = xQueueCreateStatic(
-    THERMAL_MGR_QUEUE_LENGTH, THERMAL_MGR_QUEUE_ITEM_SIZE,
-    thermalMgrQueueStorageArea, &thermalMgrQueueBuffer);
-
+      THERMAL_MGR_QUEUE_LENGTH, THERMAL_MGR_QUEUE_ITEM_SIZE,
+      thermalMgrQueueStorageArea, &thermalMgrQueueBuffer);
 }
 
-error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
+error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event)
+{
+  if (event == NULL)
+    return ERR_CODE_INVALID_ARG;
+  error_code_t errCode;
+  if (thermalMgrQueueHandle == NULL){
+    return ERR_CODE_INVALID_STATE;
+  }
+  errCode = xQueueSend(thermalMgrQueueHandle, (void *)event, 0);
+
+  if (errCode != ERR_CODE_SUCCESS)
+    return errCode;
```

Comment:
Better to use the same format for if statements. If you're using the curly braces for one of them, use it for all. I usually prefer if statements with the curly brace since it's easier to maintain.

```suggestion
  if (event == NULL) {
    return ERR_CODE_INVALID_ARG;
  }

  if (thermalMgrQueueHandle == NULL) {
    return ERR_CODE_INVALID_STATE;
  }
  
  error_code_t errCode;
  errCode = xQueueSend(thermalMgrQueueHandle, (void *)event, 0);
  if (errCode != ERR_CODE_SUCCESS) {
    return errCode;
  }
```